### DISCUSSION TITLE
Synced tab not working

### DISCUSSION THREAD

**Discussion by Andre601** [facelessuser/pymdown-extensions#2293] [Q&A] [answered]
I've tried to implement the sync between tabs as explained in the docs, but I had no luck doing so...

The code is 1 to 1 the same as in the documentation (Shown here), but no matter what I try, it does not work for me.

I'm using the tab blocks with `alternate_style: true` in MkDocs.

I also tried doing stuff like `document$.subscribe(tabSync)` to see if that works, but no luck...

Could it be that I use custom emojis (Simple icons) in the tab name using the emoji extension?

**Comment by facelessuser** [comment, marked as answer]
1. It does work fine. Here's a working example: https://codepen.io/facelessuser/pen/PoLNZJp
2. If you are using MkDocs Material, just use the built-in feature: https://squidfunk.github.io/mkdocs-material/reference/content-tabs/?h=content+link#linked-content-tabs

**Reply by Andre601** [reply]
thanks
